### DISCUSSION TITLE
Preview of delete notes

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3577] [Ideas] [answered]
Would be great to be able to preview somehow in delete notes

**Comment by zadam** [comment, marked as answer]
Do you mean like this?

<img width="525" alt="image" src="https://user-images.githubusercontent.com/617641/215899443-87ade784-3aed-49c6-b81f-fb9774d29b5b.png">

**Reply by shirobachi** [reply]
Nah, more like could got to trash can and read some deleted notes (you looking for some deleteled note)

**Reply by zadam** [reply]
I see, you can see deleted notes in the Recent changes:

<img width="437" alt="image" src="https://user-images.githubusercontent.com/617641/218040758-4adcd157-67f7-4121-8975-3e963bd32dab.png">

There's no preview, however you can undelete the notes to view them and then delete again :D

**Reply by shirobachi** [reply]
I do know that:) Just kindly suggesting adding preview :)
